src/util.jl

```diff
@@ -172,7 +173,16 @@ rms(s::AbstractArray{T}) where {T<:Number} = sqrt(sum(abs2, s)/length(s))
 # root mean square of fft of signal
 rmsfft(f::AbstractArray{T}) where {T<:Complex} = sqrt(sum(abs2, f))/length(f)
 
+function meanfreq{T<:Real}(f::AbstractVector{T},fs=2*π)
```

Comment:
Still uses the old form for parametric functions which is deprecated in Julia 0.7. As `T` isn't used in the function, this can just be
```julia
function meanfreq(f::AbstractVector{<:Real}, fs=2*π)
```
(While at it, the space before the `,` would be nice, too.)